src/components/tools/geometry-tool/geometry-tool.tsx

```diff
@@ -106,7 +106,6 @@ export default class GeometryToolComponent extends BaseComponent<IGeometryProps,
             return (
               <div className="geometry-size-me">
                 <GeometryContentComponent
-                  size={size}
```

Comment:
I'm mildly curious about this change, which is inconsistent with the CLUE branch, but it shouldn't matter for Dataflow, so it's not worth spending time tracking down if there isn't a simple answer.

Reply:
We get an error: "'size is specified more than once, so this usage will be overwritten. This spread always overwrites this property."
My assumption was that `size` is included in the spread operator when we pass `{...this.props}` into the component props.  However, I don't think this is worth much effort to clean up since ideally all of this code could/should be removed from Dataflow.

Reply:
I agree it's not worth looking into. That part of the CLUE code has been refactored since then so there's now a `GeometryContentWrapper` component and the types are defined a bit differently, so 🤷.

Reply:
The `GeometryToolComponent` (where this render is being used) is given props as type `IGeometryProps`.  `IGeometryProps` is defined as follows: `IGeometryProps = IToolTileProps & SizeMeProps;`
`SizeMeProps` does indeed contain `size`.  So when we render the `GeometryContentComponent` as follows...
```
                <GeometryContentComponent
                  size={size}
                  onSetBoard={this.handleSetBoard}
                  onSetActionHandlers={this.handleSetActionHandlers}
                  onUpdateToolbar={this.handleUpdateToolbar}
                  {...this.props} />
```
...it appears that the spread of `this.props` will indeed override the earlier `size` prop.  So maybe this has been reworked in CLUE and is no longer relevant?

Reply:
Yup, looks like the integration of the `GeometryContentComponent` has been completely reworked in CLUE and this issue is no longer relevant:
https://github.com/concord-consortium/collaborative-learning/blob/267d93990252e79ac9c927389622396db7e6a191/src/components/tools/geometry-tool/geometry-content-wrapper.tsx

Reply:
Didn't see your comment, but it looks like you already knew what I discovered with a few minutes of looking through the code.